.github/workflows/theseus-engine.yml

```
@@ -41,48 +45,186 @@ jobs:
+
+          total = len(statuses)
+          passed = sum(1 for v in statuses.values() if v == "success")
+
+          rows = "\n".join(
+              f"| {repo} | {'✅' if s == 'success' else '❌'} |"
+              for repo, s in sorted(statuses.items())
+          )
+
+          body = f"""## Automated Theseus Data Engine Run
+
+| Repo | Status |
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)